projects/ng-live-docs/src/documentation.ts

```diff
@@ -92,11 +93,31 @@ export const Documentation = {
      * Returns angular routes used when displaying the documentation/examples for the components.
      */
     getRoutes(): Routes {
-        return Documentation.getAllEntries().map((documentationEntry: DocumentationEntry) => ({
+        let routes = Documentation.getAllEntries().map((documentationEntry: DocumentationEntry) => ({
```

Comment:
We prefer using `const` when possible. It should be possible here if you just change line 119 to be `return redirectList.concat(routes)` or `return [...redirectList, ...routes]`